tests/platform_tests/api/test_fan_drawer.py

```diff
@@ -40,6 +41,11 @@ def setup(self, platform_api_conn):
             except:
                 pytest.fail("num_fan_drawers is not an integer")
 
+        chassis = duthost.facts['chassis']
```

Comment:
In case there is no platform.json file on DUT or something went wrong with reading and parsing the platform.json file, accessing `duthost.facts['chassis']` may trigger KeyError exception. It's better to use:
```
chassis = duthost.facts.get('chassis', None)
```

Reply:
fixed